evap/staff/tests/test_tools.py

```diff
@@ -219,6 +222,52 @@ def test_do_nothing_if_test_run(self):
         self.assertEqual(len(messages), 4)
 
 
+class RemoveUserDueToInactivity(TestCase):
+    def test_remove_user_due_to_inactivity(self):
+        inactive_user = baker.make(UserProfile, last_name="Inactive User")
+        inactive_evaluation = baker.make(
+            Evaluation,
+            state=Evaluation.State.PUBLISHED,
+            vote_start_datetime=datetime.today()
+            - timedelta(days=settings.PARTICIPATION_DELETION_AFTER_INACTIVE_MONTHS + 1),
+            vote_end_date=datetime.today() - timedelta(days=settings.PARTICIPATION_DELETION_AFTER_INACTIVE_MONTHS + 1),
+            participants=[inactive_user],
+        )
+
+        semester = inactive_evaluation.course.semester
+        semester.archive()
+
+        messages = remove_inactive_participations(inactive_user)
+        self.assertTrue(inactive_user.can_be_marked_inactive_by_manager)
+        self.assertEqual(len(messages), 1)
```

Comment:
I think we should additionally assert what the content of the message is, i.e.
```python3
self.assertEqual(messages, [f"{user.full_name} will be removed from 1 participation(s) due to inactivity"])
```
to make the test more assertive. Same for the other test that asserts that len == 1 below.